packages/notification-center/rollup.config.js

```diff
@@ -32,6 +34,29 @@ export default [
       image(),
     ],
   },
+  {
+    input: 'src/web-component.ts',
+    external: ['react@17.0.2', 'react-dom@17.0.2'],
+    output: [
+      {
+        file: packageJson.browser,
+        format: 'umd',
+        name: 'NotificationCenterWebComponent',
+        sourcemap: true,
+      },
```

Comment:
the UMD module rollup configuration, the file will be built to the path from `package.json` - `browser` param